CHANGELOG.rst

```diff
@@ -32,38 +35,103 @@ Added Features
   (`#2596 <https://github.com/girder/girder/pull/2596>`_)
 * Added the server FUSE plugin that mounts Girder files in a read-only
   user-space filesystem. (`#2521 <https://github.com/girder/girder/pull/2521>`_)
+* Plugins can be enabled for Pytest. (`#2634 <https://github.com/girder/girder/pull/2634>`_)
```

Comment:
Should we move this to a subelement of the pytest entry (below)?  And, should this and other test-related features get moved to the Testing section?

Reply:
Yeah, could you add a commit?